src/com/owncloud/android/operations/UpdateOCVersionOperation.java

```diff
@@ -69,9 +69,9 @@ protected RemoteOperationResult run(OwnCloudClient client) {
             get = new GetMethod(statUrl);
             int status = client.executeMethod(get);
             if (status != HttpStatus.SC_OK) {
+                result = new RemoteOperationResult(false, get);
```

Comment:
This "get" variable name is too generic, could you please change the name to a more specific one?